webapp/app/forms/flows/lotse_flow.py

```diff
@@ -354,7 +358,7 @@ def _collect_data_for_step(self, step, form_data, missing_fields=None):
                 field = getattr(step.form, attr)
                 label, value = self._generate_value_representation(field, form_data[attr])
                 if value:
-                    step_data[label] = value
+                    step_data[label] = step._get_value_representation(value)
```

Comment:
If we use this function from the outside, why make it private?

Reply:
sry outdated - it's not private anymore :)